<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageNotes</class>
 <widget class="QDialog" name="ManageNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion notes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="ManageNotesList" name="note_list">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="first_name_label">
       <property name="text">
        <string>Prénom: </string>
       </property>
       <property name="buddy">
        <cstring>first_name_input</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="birthdate_label">
       <property name="text">
        <string>Date de naissance: </string>
       </property>
       <property name="buddy">
        <cstring>birthdate_input</cstring>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="3">
      <widget class="QLabel" name="promo_label">
       <property name="text">
        <string>Promo:</string>
       </property>
       <property name="buddy">
        <cstring>promo_input</cstring>
       </property>
      </widget>
     </item>
     <item row="22" column="0" colspan="3">
      <widget class="QLabel" name="photo_label">
       <property name="text">
        <string>Photo:</string>
       </property>
       <property name="buddy">
        <cstring>photo_button</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="Input" name="nickname_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="33" column="1" colspan="2">
      <widget class="QPushButton" name="save_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Sauvegarder</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="Input" name="first_name_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="28" column="0" colspan="3">
      <widget class="QListWidget" name="category_list"/>
     </item>
     <item row="29" column="1">
      <widget class="QPushButton" name="add_category_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ajouter à la catégorie</string>
       </property>
      </widget>
     </item>
     <item row="29" column="2">
      <widget class="QPushButton" name="remove_category_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Enlever de la catégorie</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="QLabel" name="mail_label">
       <property name="text">
        <string>Mail:</string>
       </property>
       <property name="buddy">
        <cstring>mail_input</cstring>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="3">
      <widget class="Input" name="phone_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Nom:</string>
       </property>
       <property name="buddy">
        <cstring>name_input</cstring>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="3">
      <widget class="QComboBox" name="promo_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>1A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Externe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ESIAB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prof</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ancien</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="33" column="0">
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="3">
      <widget class="Input" name="birthdate_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="3">
      <widget class="Input" name="mail_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="29" column="0">
      <widget class="QComboBox" name="category_selector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="Input" name="name_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="20" column="0" colspan="2">
      <widget class="QLabel" name="phone_label">
       <property name="text">
        <string>Téléphone:</string>
       </property>
       <property name="buddy">
        <cstring>phone_input</cstring>
       </property>
      </widget>
     </item>
     <item row="25" column="0" colspan="2">
      <widget class="QLabel" name="photo">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>160</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="nickname_label">
       <property name="text">
        <string>Surnom: </string>
       </property>
       <property name="buddy">
        <cstring>nickname_input</cstring>
       </property>
      </widget>
     </item>
     <item row="24" column="1" colspan="2">
      <widget class="QCheckBox" name="stats_checkbox">
       <property name="text">
        <string>Stats</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="23" column="0" colspan="3">
      <widget class="QPushButton" name="photo_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Modifier la photo</string>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QCheckBox" name="mails_checkbox">
       <property name="text">
        <string>Mails</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="26" column="0" colspan="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="27" column="0" colspan="3">
      <widget class="QPushButton" name="cotiz_button">
       <property name="text">
        <string>Cotiz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QLineEdit</extends>
   <header>gui/input_widget</header>
  </customwidget>
  <customwidget>
   <class>ManageNotesList</class>
   <extends>QListWidget</extends>
   <header>gui/notes_management_window</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nickname_input</tabstop>
  <tabstop>first_name_input</tabstop>
  <tabstop>name_input</tabstop>
  <tabstop>birthdate_input</tabstop>
  <tabstop>promo_input</tabstop>
  <tabstop>mail_input</tabstop>
  <tabstop>phone_input</tabstop>
  <tabstop>photo_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>photo_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>add_photo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>add_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>save_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>note_list</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>ManageNotes</receiver>
   <slot>on_note_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>del_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_category_button</sender>
   <signal>pressed()</signal>
   <receiver>ManageNotes</receiver>
   <slot>add_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_category_button</sender>
   <signal>pressed()</signal>
   <receiver>ManageNotes</receiver>
   <slot>remove_category_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cotiz_button</sender>
   <signal>clicked()</signal>
   <receiver>ManageNotes</receiver>
   <slot>cotiz_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_photo()</slot>
  <slot>add_fnc()</slot>
  <slot>save_fnc()</slot>
  <slot>on_note_selected()</slot>
  <slot>del_fnc()</slot>
  <slot>add_category_fnc()</slot>
  <slot>remove_category_fnc()</slot>
  <slot>cotiz_fnc()</slot>
 </slots>
</ui>
